package.json

```diff
@@ -56,7 +56,7 @@
   "bugs": "https://github.com/eslint/eslint/issues/",
   "dependencies": {
     "@eslint/eslintrc": "^1.3.3",
-    "@humanwhocodes/config-array": "^0.11.6",
+    "@humanwhocodes/config-array": "0.11.7",
```

Comment:
Can we keep the caret?

Reply:
This is to entirely fix CI at this moment, as some tests are failing with 0.11.8. Will be fixed by https://github.com/eslint/eslint/pull/16579.